### ISSUE TITLE
Adding new limiters?

### ISSUE THREAD

**Issue by donnaaboise** [clawpack/amrclaw#239]
Are there any thoughts about how to best add a new limiter?  I am thinking of a case in which the user adds a new limiter to `inline limiter.f90`.  There doesn't seem to be any way to indicate to the Python routines that they should accept a new limiter number.    One idea is to allow a `'user'` option + a number, i.e.  The number would then indicate the goto location in the limiter file (or at least an offset from limiters known to Clawpack).   The user is responsible for making sure this limiter gets called correctly (since they are supplying  new limiter file).

**Comment by mjberger** [comment]
What brought this up? Do you want to add a new limiter yourself or know somebody who does? How interesting.
 — Marsha

**Comment by mandli** [comment]
If you want to just test a limiter you could use PyClaw which has a number of additional limiters and would be trivial to add a new one.  Otherwise in `clawutil/src/python/clawutil/data.py` starting at line 733 you could add another limiter option along with the associated code for the limiter in Fortran.

**Comment by donnaaboise** [comment]
Yes, I'd like to try out a new limiter. I am comparing results with Basilisk, and they use the [generalized minmod limiter](https://en.wikipedia.org/wiki/Flux_limiter#Generalised_minmod_limiter).  I am not necessarily proposing to add it to Clawpack, though, and so haven't opted to edit `data.py` (as per suggestion by @mandli).  In the same way we include a user option for boundary conditions, I just thought it would be useful to have a user option for limiters.   Right now, I have to manually edit `claw.data` to include a number outside of [0,4].

**Comment by mandli** [comment]
It would be pretty easy to add.  One thing we did do in PyClaw was add another argument for the courant number as some of the more exotic limiters out there required the courant number.  You can see the list if you are interested in https://github.com/clawpack/pyclaw/blob/master/src/pyclaw/limiters/tvd.py.
